test/functional/test-url-replacements.js

```diff
@@ -455,26 +455,11 @@ describes.sandboxed('UrlReplacements', {}, () => {
           .expandUrlAsync('?url=SOURCE_URL')
           .then(res => {
             expect(res).to.equal(
-                '?url=' + encodeURIComponent('http://o.com/foo/?hello=world&safe=1&a=1'));
+                '?url=' + encodeURIComponent('http://o.com/foo/?a=1&safe=1&hello=world'));
```

Comment:
the description of this test case is a bit mis-leading. 
should ignore extra params that already exists